client/controllers/AddToPlanningController.tsx

```diff
@@ -197,20 +217,14 @@ export class AddToPlanningController {
         contentProfile: IContentProfile
     }> {
         return this.api.find('archive', this.item._id)
-            .then((newsItem: IArticle) => (
-                superdeskApi.entities.contentProfile.get(newsItem.profile)
-                    .then((contentProfile) => ({
-                        newsItem,
-                        contentProfile,
-                    }))
-                    .catch((error) => {
-                        this.notify.error(
-                            getErrorMessage(error, this.gettext('Failed to load content profile.'))
-                        );
-                        this.$scope.resolve(error);
-                        return Promise.reject(error);
-                    })
-            ), (error) => {
+            .then((newsItem: IArticle) => {
+                const contentProfile = superdeskApi.entities.contentProfile.get(newsItem.profile);
+
+                return Promise.resolve({
```

Comment:
no need for `Promise.resolve` here. You should return the object directly. Since it's inside a `.then` it's a promise already